spec.html

```diff
@@ -23704,16 +23718,10 @@ <h1>Runtime Semantics: Encode ( _string_, _unescapedSet_ )</h1>
                 1. Let _S_ be the String value containing only the code unit _C_.
                 1. Set _R_ to the string-concatenation of the previous value of _R_ and _S_.
               1. Else,
-                1. If _C_ is a <emu-xref href="#trailing-surrogate"></emu-xref>, throw a *URIError* exception.
-                1. If _C_ is not a <emu-xref href="#leading-surrogate"></emu-xref>, then
-                  1. Let _V_ be the code point with the same numeric value as code unit _C_.
-                1. Else,
-                  1. Increase _k_ by 1.
-                  1. If _k_ equals _strLen_, throw a *URIError* exception.
-                  1. Let _kChar_ be the code unit at index _k_ within _string_.
-                  1. If _kChar_ is not a <emu-xref href="#trailing-surrogate"></emu-xref>, throw a *URIError* exception.
-                  1. Let _V_ be UTF16Decode(_C_, _kChar_).
-                1. Let _Octets_ be the List of octets resulting by applying the UTF-8 transformation to _V_.
+                1. Let _cp_ be ! CodePointAt(_string_, _k_).
+                1. If _cp_ has the same numeric value as a <emu-xref href="#leading-surrogate"></emu-xref> or <emu-xref href="#trailing-surrogate"></emu-xref>, throw a *URIError* exception.
+                1. If the numeric value of _cp_ &gt; 0xFFFF, increase _k_ by 1.
```

Comment:
We mix our terminology with "increase _k_ by 1" and "increment _k_ [by 1]" throughout the spec. @tc39/ecma262-editors should we normalise this phrasing? I prefer "increment *k*" (dropping the "by *n*" or "*n* times" for the case where *n* is 1).

Reply:
Let’s use whatever’s most common in this PR; please file a separate issue to track overall normalization.

Reply:
(See #1266 for some related discussion.)